contracts/ccip/GovernanceCCIPReceiver.sol

```diff
@@ -3,81 +3,55 @@ pragma solidity ^0.8.20;
 
 import {Client} from "@chainlink/contracts-ccip/src/v0.8/ccip/libraries/Client.sol";
 import {CCIPReceiver} from "@chainlink/contracts-ccip/src/v0.8/ccip/applications/CCIPReceiver.sol";
-import {Test, console} from "forge-std/Test.sol";
+import {IGovernanceCCIPReceiver} from "./interfaces/IGovernanceCCIPReceiver.sol";
 
 /**
  * @title GovernanceReceiver
  * @dev A contract for receiving and executing governance proposals from Ethereum Mainnet via CCIP.
  * This contract processes cross-chain messages and executes them on the destination chain.
  */
-contract GovernanceCCIPReceiver is CCIPReceiver {
-  /// @notice The address of the sender contract on Ethereum Mainnet.
-  address public mainnetSender;
+contract GovernanceCCIPReceiver is IGovernanceCCIPReceiver, CCIPReceiver {
+  /// @inheritdoc IGovernanceCCIPReceiver
+  address public immutable mainnetSender;
 
-  /// @notice The chain selector for Ethereum Mainnet.
-  uint64 public mainnetChainSelector;
+  /// @inheritdoc IGovernanceCCIPReceiver
+  uint64 public constant mainnetChainSelector = 5009297550715157269;
```

Comment:
@voith this is a constant, replace the name in **uppercase** following the Solidity style recommendations